src/webapp/components/survey-list/table/PaginatedSurveyListTable.tsx

```diff
@@ -47,6 +51,14 @@ export const PaginatedSurveyListTable: React.FC<PaginatedSurveyListTableProps> =
     //states for column sort
     const [surveyNameSortDirection, setSurveyNameSortDirection] = useState<SortDirection>("asc");
     const [patientIdSortDirection, setPatientIdSortDirection] = useState<SortDirection>("asc");
+    const [sampleShipmentsSortDirection, setSampleShipmentsSortDirection] =
```

Comment:
move all the child column sorting states, logic and onclick handler to a customHook called useMultipleChilCount or something similar